tls/s2n_quic_support.h

```diff
@@ -73,4 +73,4 @@ typedef int (*s2n_secret_cb) (void* context, struct s2n_connection *conn,
  * The callback function will ONLY be triggered if QUIC is enabled. This API is not intended to be
  * used outside of a QUIC implementation.
  */
-int s2n_connection_set_secret_callback(struct s2n_connection *conn, s2n_secret_cb cb_func, void *ctx);
+S2N_API int s2n_connection_set_secret_callback(struct s2n_connection *conn, s2n_secret_cb cb_func, void *ctx);
```

Comment:
Shouldn't we be adding  `S2N_API`  within `api/s2n.h` file?
I notice that the ` tls/s2n_quic_support.h` APIs are not in `api/s2n.h` like our other APIs.

Reply:
The idea is it's currently an opt-in feature, so we don't include these APIs by default.